flake.nix

```diff
@@ -6,7 +6,26 @@
     };
     nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
     cardano-node.url = "github:input-output-hk/cardano-node?rev=73f9a746362695dc2cb63ba757fbcabb81733d23";
-    cardano-transaction-lib.url = "github:Plutonomicon/cardano-transaction-lib?rev=3ccbfde6fc9728c6b1b3e22ec49247716fcbf689";
+    # CTL with cardano-wallet fix
+    cardano-transaction-lib = {
+      url = "github:Plutonomicon/cardano-transaction-lib?rev=296563fa1a9a7494393bb4a74ea0464054fe9b4b";
```

Comment:
Yeah sorry about that, it was hard this time because it involved a lot of separate small updates to ctl to pickup features and fixes as they were happening.